xnmt/translator.py

```diff
@@ -262,8 +269,16 @@ def __call__(self, translator, dec_state, src, trg):
       done = list(six.moves.map(lambda x: x == Vocab.ES, sample))
       if all(done):
         break
+
+    if self.use_baseline:
+      h_t = dy.tanh(translator.decoder.context_projector(dy.concatenate([dec_state.rnn_state.output(), dec_state.context])))
+      if not self.baseline:
+        model = translator.decoder.param_col
```

Comment:
Initiate this in the constructor

Reply:
Find a better way to pass the information about the size of the input. But this should be done in the constructor I think.